src/main/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -54,6 +54,33 @@
     private final OverlapDetector<Chain> chains;
     private final Map<String, Set<String>> contigMap = new HashMap<>();
 
+    private boolean logFailedIntervals = true;
+    private long totalFailedIntervalsBelowThreshold = 0L;
+
+    /**
+     * By default any lifted interval that falls below liftOverMinMatch
+     * will be logged.  Set this to false to prevent logging.
+     * @param logFailedIntervals
+     */
+    public void setLogFailedIntervals(boolean logFailedIntervals) {
+        this.logFailedIntervals = logFailedIntervals;
+    }
+
+    /**
+     * Resets the internal counter that tracks intervals that failed liftover due to insufficient intersection length
+     */
+    public void resetFailedIntervalCounter(){
```

Comment:
here and below, please put a space between the `()` and the `{`. 

also after the condition in an `if` statement:
```
if (condition)␣{
     expression
}
```

Reply:
OK.  4 instances have been replaced.